tests/bluetooth/bsim_bt/bsim_test_advx/src/main.c

```diff
@@ -188,6 +188,15 @@ static void test_advx_main(void)
 	}
 	printk("success.\n");
 
+	k_sleep(K_MSEC(400));
+
+	printk("Removing adv aux set...");
+	err = ll_adv_aux_set_remove(handle);
+	if (err) {
+		goto exit;
+	}
+	printk("success.\n");
+
```

Comment:
Missing tests to check if adv set can be created and used.

Is the bsim test passing? does line 194, 200, and 206 succeed after adv set is removed?

Reply:
Modified bsim test. It is passing.